modules/osdk-helm-project-layout.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/helm/osdk-helm-project-layout.adoc
+
+[id="osdk-helm-project-layout_{context}"]
+= Helm-based project layout
+
+Helm-based Operator projects generated using the `operator-sdk init --plugins helm` command contain the following directories and files:
+
+[options="header",cols="1,4"]
+|===
+
+|File/folders |Purpose
+
+|`config`
+|Kustomize manifests for deploying the Operator on a Kubernetes cluster.
```

Comment:
Customize

Reply:
Nevermind. TIL Kustomize is a thing!

Reply:
Added link here to https://kustomize.io just in case!